curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-dom-manipulation-by-building-a-role-playing-game/62a3b79d520a7f3d0e25afd6.md

```diff
@@ -1,30 +1,36 @@
 ---
 id: 62a3b79d520a7f3d0e25afd6
-title: Step 19
+title: Step 14
 challengeType: 0
-dashedName: step-19
+dashedName: step-14
 ---
 
 # --description--
 
 JavaScript interacts with the HTML using the <dfn>Document Object Model</dfn>, or DOM. The DOM is a tree of objects that represents the HTML. You can access the HTML using the `document` object, which represents your entire HTML document.
 
-One method for finding specific elements in your HTML is using the `querySelector()` method. The `querySelector()` method takes a CSS selector as an argument and returns the first element that matches that selector. For example, to find the `<h1>` element in your HTML, you would write:
+One method for finding specific elements in your HTML is using the `querySelector()` method. The `querySelector()` method takes a CSS selector as an argument and returns the first element that matches that selector. 
+
+Here is an example of using `querySelector()` to find an element with the `id` of `example`:
+
+```html
+<div id="example"></div>
+```
 
 ```js
-let h1 = document.querySelector("h1");
+let exampleDiv = document.querySelector("#example");
 ```
 
-Note that `h1` is a string and matches the CSS selector you would use.
+Note that `#example` is a string and matches the CSS selector you would use.
```

Comment:
wouldn't it be nice if we told the users that thet can use `.` to target classname, `#` to target id and the tag name to target tag names ? @jdwilkin4

Reply:
I vote we leave this particular sentence as is. Reason being, is because with the current version, we still have campers confused on what the syntax is for the challenge. So explicitly showing them `#example` will match what they will have to write in the challenge.

We could add another sentence, before this explaining that they could use classes, ids, tag names, etc for the querySelector.